app/controllers/events_controller.rb

```diff
@@ -1,3 +1,7 @@
 class EventsController < ApplicationController
-  def index; end
+  #This method will show the events in /events view
+  def events
```

Comment:
Remove the comment `#This method will show the events in /events view` as it is redundant and does not provide any additional information.